src/locale/locales/fr/messages.po

```diff
@@ -461,7 +431,7 @@ msgstr "Bluesky"
 
 #: src/view/com/auth/server-input/index.tsx:150
 msgid "Bluesky is an open network where you can choose your hosting provider. Custom hosting is now available in beta for developers."
-msgstr ""
+msgstr "Bluesky est un réseau ouvert où vous pouvez choisir votre fournisseur d’hébergement. L’hébergement personnalisé est désormais disponible en version bêta pour les développeurs."
```

Comment:
We settled elsewhere in `hébergeur` for `hosting provider`: it's shorter and clearer in French.

`auto-hébergement` (literally `self hosting`) is [a commonly used](https://fr.wikipedia.org/wiki/Auto-h%C3%A9bergement_(Internet)) for what is referred to `hébergement personnalisé` here.

```suggestion
msgstr "Bluesky est un réseau ouvert où vous pouvez choisir votre hébergeur. L’auto-hébergement est désormais disponible en version bêta pour les développeurs."
```